.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -58,6 +58,34 @@ jobs:
           commit_message: 🤖 Bump ${{ github.event.inputs.release-type }} version of Airbyte CDK
           commit_user_name: Octavia Squidington III
           commit_user_email: octavia-squidington-iii@users.noreply.github.com
+      - name: Post failure to Slack channel dev-connectors-extensibility
```

Comment:
My main worry about posting this to our main channel is that we might start to spam or clutter it if we have a lot of deploys or failed workflows in any given day. How would you feel about a side channel #airbyte-cdk-releases or something like that?

Reply:
sounds good thank you. if alerts ultimately are not too noisy i have no concerns moving it to main channel.